tests/test.js

```diff
@@ -136,10 +136,10 @@ describe('roundNumber(num)', function() {
 });
 
 describe('roundUp(num)', function() {
-  it('should return the argument after squaring it', function() {
-    expect(exercises.roundUp(1.5)).toBe(2);
+  it('should return the argument after rounding Up', function() {
```

Comment:
I think this is the only thing that needs to be changed, because they want us to round UP, no matter what. I think we're supposed to use Math.ceil. :)